README.md

```diff
@@ -1,2 +1,52 @@
-# design-cfps
-Repo to store Cilium CFP design docs
+This directory contains CFP design proposals for features impacting repos across the
+Cilium Github organization.
+
+# Purpose of CFPs
+
+The purpose of a Cilium Feature Proposal (CFP) is to allow community members to gain feedback
+on their designs from the Committers before the community member commits to
+executing on the design. By going through the design process, developers gain a
+high level of confidence that their designs are viable and will be
+accepted.
+
+NOTE: This process is not mandatory. Anyone can execute on their own design
+without going through this process and submit code to the respective repos.
+However, depending on the complexity of the design and how experienced the
+developer is within the community, they could greatly benefit from going through
+this design process first. The risk of not getting a design proposal approved
+is that a developer may not arrive at a viable architecture that the community will
+accept.
+
+# How to create CFPs
+
+To create a CFP, it is recommended to use the `CFP-001-template.md`
+file as an outline. The structure of this template is meant to provide a starting
+point for people. Feel free to edit and modify your outline to best fit your
+needs when creating a proposal. The title should be `CFP-###-subject.md` 
+where the number is the PR number.
+
+Many design docs also begin their life as a Google doc or other shareable
+file for easy commenting and editing when still in the early stages of discussion.
+Once your proposal is done, submit it as a PR to the design-cfps folder.
```

Comment:
```suggestion
Once your proposal is done, submit it as a PR to the design-cfps repository.
```

By the way, do you want the CFPs to be added to the root of this repo, or in a subdir?
[EDIT] OK never mind, I see that the next commit adds the subdir.